app/assets/javascripts/miq_tree.js

```diff
@@ -394,21 +394,21 @@ function miqSquashToggle(treeName) {
 
 function miqTreeEventSafeEval(func) {
   var whitelist = [
-    'miqOnCheckCUFilters',
-    'miqOnCheckGenealogy',
+    'miqOnCheckCUFilters', // Settings -> Region -> CU Collection tab
+    'miqOnCheckGenealogy', // Compute -> Ingrastructure -> VMs -> Select one vm and click on genealogy
     'miqOnCheckGeneric',
-    'miqOnClickMenuRoles',
-    'miqOnCheckProtect',
-    'miqOnCheckSections',
+    'miqOnClickMenuRoles', // Settings -> Access Controll -> Roles/Edit Roles // Seems like not doing anything
+    'miqOnCheckProtect', // Compute -> Infra -> VMs -> Select VM -> Policy button -> edit
+    'miqOnCheckSections', // Compute -> Infra -> VMs -> Compare VM's
     'miqOnCheckUserFilters',
-    'miqOnClickAutomate',
-    'miqOnClickAutomateCatalog',
-    'miqOnClickDiagnostics',
-    'miqOnClickGeneric',
-    'miqOnClickHostNet',
-    'miqOnClickSnapshots',
-    'miqOnClickUtilization',
-    'miqOnCheckTenantTree',
+    'miqOnClickAutomate', // Atuomate -> Atuomate // Maybe not used
```

Comment:
This one is used when copying an automate method.

Reply:
@pkomanek could you help us by describing a scenario how can we get to this tree?

Reply:
Found it by your description: Automate -> Expoler -> Select a class -> Copy -> Uncheck `Copy to same path` -> Namespace selection uses thath tree